ipatests/test_integration/test_acme.py

```diff
@@ -769,29 +824,213 @@ def test_pruning_negative_options(self):
                  cmd, '--enable'],
                 raiseonerr=False
             )
-            assert result.returncode == 1
+            assert result.returncode == 2
             assert "Cannot change and show config" in result.stderr_text
 
         result = self.master.run_command(
             ['ipa-acme-manage', 'pruning',
-             '--cron="* *"'],
+             '--cron=* *'],
             raiseonerr=False
         )
-        assert result.returncode == 1
-        assert "Invalid format format --cron" in result.stderr_text
+        assert result.returncode == 2
+        assert "Invalid format for --cron" in result.stderr_text
 
         result = self.master.run_command(
             ['ipa-acme-manage', 'pruning',
-             '--cron="100 * * * *"'],
+             '--cron=100 * * * *'],
             raiseonerr=False
         )
         assert result.returncode == 1
         assert "100 not within the range 0-59" in result.stderr_text
 
         result = self.master.run_command(
             ['ipa-acme-manage', 'pruning',
-             '--cron="10 1-5 * * *"'],
+             '--cron=10 1-5 * * *'],
             raiseonerr=False
         )
         assert result.returncode == 1
         assert "1-5 ranges are not supported" in result.stderr_text
+
+    def test_prune_cert_manual(self, issue_and_expire_acme_cert):
+        """Test to prune expired certificate by manual run"""
+        if (tasks.get_pki_version(self.master)
+           < tasks.parse_version('11.3.0')):
+            raise pytest.skip("Certificate pruning is not available")
+
+        issue_and_expire_acme_cert(
+            self.master, self.clients[0], self.acme_server)
+
+        # check that the certificate issued for the client
+        result = self.master.run_command(['ipa', 'cert-find'])
+        assert 'CN=%s' % self.clients[0].hostname in result.stdout_text
+
+        # run prune command manually
+        self.master.run_command(['ipa-acme-manage', 'pruning', '--enable'])
+        self.master.run_command(['ipactl', 'restart'])
+        self.master.run_command(['ipa-acme-manage', 'pruning', '--run'])
+        time.sleep(100)
```

Comment:
I assume we'll be able to narrow this done eventually. I think the job is triggered pretty quickly. But a comment might be in order to explain. X seconds for the tool to prep for the run + Y seconds for the job to complete + Z for cosmic rays. I have the feeling this can be more like 20 seconds but lets get some successful runs to validate it.